packages/react-menu/e2e/MenuList.e2e.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { mount as mountBase } from '@cypress/react';
+
+import { FluentProvider } from '@fluentui/react-provider';
+import { teamsLightTheme } from '@fluentui/react-theme';
+
+import { menuTriggerSelector, menuItemSelector, menuSelector } from './selectors';
+
+import { MenuList, MenuItem, Menu, MenuTrigger, MenuPopover } from '@fluentui/react-menu';
+const mount = (element: JSX.Element) => {
```

Comment:
I do think that a custom mount should be shared, but I'm not sure where it could actually live, any proposals welcome

Reply:
TMP style ? yeah sure

Reply:
- we should boot new package for cypress related things.

- I'd suggest the API interface look like following:

```ts
import {type MountOptions, mount as cyMount} from '@cypress/react';

const Root = (props:{children:React.ReactElement}) => <FluentProvider theme={teamsLightTheme}>{props.children}</FluentProvider>;

interface Options extends MountOptions {
  root?: typeof Root
}

export function mount(markup: React.ReactElement, options: Options ):  ReturnType<typeof cyMount> {
   const {root = Root, ...cyOptions} = options;
    return cyMount(<root>{markup}</root>,cyOptions)
}
```

Reply:
yes, you're asking how to approach this, I provided answer :D sounds good ty!